package.json

```diff
@@ -18,6 +18,7 @@
     "format": "prettier --write \"{apps,packages}/**/*.{ts,tsx,md,css,scss}\""
   },
   "devDependencies": {
+    "@csstools/postcss-oklab-function": "4.0.8",
```

Comment:
I don't see this one being used anywhere, can it be removed? It probably should not be in root package.json either.